### ISSUE TITLE
add space between fields in password setting

### ISSUE THREAD

**Issue by rlecellier** [ecreall/nova-ideo#8]
From registration mail, the fields to set the password are sticked.  A small space between them would be nice.

**Comment by vincentfretin** [comment]
You're right! And a sentence explaining what to do would be nice: "Please choose a password to confirm your subscription".

**Comment by vincentfretin** [comment]
The sentence was added in https://github.com/ecreall/nova-ideo/commit/9201aed4f1e589a2da1965daa231ab36b2a4707c
We still need to add some space between the two input.
